Task6/Task6.UI/Program.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Task6.Entities;
+using Task6.Serializers;
+
+var catalog = new Catalog();
+
+var dostoevsky = new Author("Fyodor", "Dostoevsky", new DateTime());
+var backman = new Author("Frederik", "Backman", new DateTime());
+var hoover = new Author("Colleen", "Hoover", new DateTime());
+
+var book1 = new Book("Crime and Punishment", new DateTime(1866, 1, 1), new HashSet<Author> { dostoevsky });
+var book2 = new Book("The Idiot", new DateTime(1868, 1, 1), new HashSet<Author> { dostoevsky });
+var book3 = new Book("Beartown", new DateTime(2016, 1, 1), new HashSet<Author> { backman, hoover });
+var book4 = new Book("Us Against You", new DateTime(2017, 1, 1), new HashSet<Author> { backman });
+
+catalog.AddBook("978-0-59-652068-7", book1);
+catalog.AddBook("978-3-16-148410-0", book2);
+catalog.AddBook("9789880038273", book3);
+catalog.AddBook("1566199093257", book4);
+
```

Comment:
Seems that this not a full task solution ->
4	Create and fill the Catalog book by real books with authors and isbn (not less then 4 books, some books has to have several authors, author has to have several books)
5	Save catalog object to a xml file (use XMLSerializer class). See sample of xml file.
6	Restore Catalog object from xml file. Check identity to initial object.
7	Save catalog object into json files (one file-one author with all his books.)
8	Restore catalog object from json files. Check identity.